appintro/src/main/res/values/colors.xml

```diff
@@ -1,4 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources xmlns:tools="http://schemas.android.com/tools">
-    <color name="appintro_default_background_color">#222222</color>
+<resources>
+    <color name="appintro_background_color">#009688</color>
```

Comment:
Please revert to `#222222` as everyone that haven't set a custom color will see it's onboarding changed from dark gray to green xD

Reply:
Done